site/includes/breadcrumbscontent.hbs

```diff
@@ -0,0 +1,25 @@
+<li>
+	<a href="http://wet-boew.github.io/wet-boew/index-{{language}}.html">{{{i18n "tmpl-home"}}}</a>
```

Comment:
IMO the current structure is pretty decent.

The breadcrumbs in the style guide and most of WET's themes (minus GCWeb) currently start off with a "Home" link that points to WET's main home page, followed by a link to their own repo's index pages and so on. That helps keep the demo sites interconnected.